odoo/addons/base/models/res_users.py

```diff
@@ -338,7 +338,6 @@ def _get_default_image(self):
                                  compute='_compute_accesses_count', compute_sudo=True)
     groups_count = fields.Integer('# Groups', help='Number of groups that apply to the current user',
                                   compute='_compute_accesses_count', compute_sudo=True)
-    image_1920 = fields.Image(related='partner_id.image_1920', inherited=True, readonly=False, default=_get_default_image)
```

Comment:
Why is it fine to remove this? Was was the purpose of this line, change the default? Maybe you can also remove the `_get_default_image` method from user?

Reply:
Since image_1920 is inherited from image.mixin/avatar.mixin, we don't need to have it. It was there before because of the default method, but we don't use it anymore

Reply:
You should still delete the default method then.